src/lib/mocha.ts

```diff
@@ -62,5 +60,8 @@ export async function runMocha(
 ): Promise<number> {
 	const mocha = createMochaInstance(options);
 	mocha.files = ['/main.js'];
+	// Call with true tell mocha that we will be handling the file load outselves
+	mocha.lazyLoadFiles(true);
```

Comment:
Let's move this inside `loadFilesAsync` like this:

https://github.com/mochajs/mocha/blob/68308215a3483504cf3614915c26ff890990846c/lib/mocha.js#L459